### ISSUE TITLE
Provide code cannot running

### ISSUE THREAD

**Issue by wintheworld** [icedle/CBPH-Net#2]
![image](https://github.com/icedle/CBPH-Net/assets/11472344/83f3881d-ade0-49d1-97cb-20a4226402d3)
LayerNorm_s function is not provided, so why you can run this code and publish?

**Comment by icedle** [comment]
Thank you for figuring out the mistake. Before push the code on the Github, I refined the code because I want the readers can
find the key part of the code quickly, so i take "CBPH.py" and "CA.py" out of model/common.py, but i forget class LayerNorm_s

class LayerNorm_s(nn.Module):

    def __init__(self, normalized_shape, eps=1e-6, data_format="channels_last"):
        super().__init__()
        self.weight = nn.Parameter(torch.ones(normalized_shape))
        self.bias = nn.Parameter(torch.zeros(normalized_shape))
        self.eps = eps
        self.data_format = data_format
        if self.data_format not in ["channels_last", "channels_first"]:
            raise NotImplementedError
        self.normalized_shape = (normalized_shape,)

    def forward(self, x):
        if self.data_format == "channels_last":
            return F.layer_norm(x, self.normalized_shape, self.weight, self.bias, self.eps)
        elif self.data_format == "channels_first":
            u = x.mean(1, keepdim=True)
            s = (x - u).pow(2).mean(1, keepdim=True)
            x = (x - u) / torch.sqrt(s + self.eps)
            x = self.weight[:, None, None] * x + self.bias[:, None, None]
            return x

The code has other parts also need to change, I have send the possible documents through the email and ensure that the code can run (I don't know whether you are the same people). 

The original code still need some time because I have return home and need to take time for garduate thesis.

At last, the paper can be published because I took long time to build the dataset, code, do the experiments, write the paper, response to the comments proposed by reviewers,  please don't ask such silly and funny question.

**Comment by wintheworld** [comment]
我已经接收您的邮件

**Comment by xiaoli440** [comment]
完整的代码文件可以发送我一份吗
